test/unit/data/prediction_jobs.json

```diff
@@ -35,6 +35,80 @@
             0.7,
             0.9,
             0.95
+        ],
+        "hyperparams" : {
+            "subsample" : 0.9650102355823993,
+            "min_child_weight": 3,
+            "max_depth": 6,
+            "gamma": 0.1313691782115394,
+            "colsample_bytree": 0.8206844265155975,
+            "silent": 1,
+            "objective": "reg:squarederror",
+            "eta": 0.010025843216782565,
+            "training_period_days": 90
+        },
+        "features_set" : [
```

Comment:
I m not at all sure if we have consistency in naming. Do we know what we mean by `feature`,  `features`, `feature_set`, `featureset`, `featuresets`, `featureset_name`. We should make sure that we at least use the same name for the same thing. Probably good to double check.

Reply:
`featureset_name`  was the "letter encoding" (from A to G) for feature sets. I think we don't use it anymore. We now use `features_set` which is the list of the names of features, appearing in the predition job. There are some `featuresets` left in optimize_hyperparameters. I didn't process this file yet.

Reply:
I looks like `features` is often used when referring to a list of features name. Perhaps its nice to use that here as well?

Reply:
I think it's good to make a distinction between `features_set` and `features_sets` in case we have several sets of features. If the case does not happen, then just `features` sounds good to me.